### DISCUSSION TITLE
Adding disable left click search to the bank plugin

### DISCUSSION THREAD

**Discussion by bibbypack** [runelite/runelite#16628] [Ideas]
Currently while doing bankstanding skills you might missclick on the search button when trying to withdraw items from the most bottom right bank slot.
Would be nice to see a feature much like the disable left click bank invent/bank equipment for the search option.

**Comment by Alexsuperfly** [comment]
the bank plugin has options to do this

**Reply by bibbypack** [reply]
![ai99L9k](https://github.com/runelite/runelite/assets/134688116/01d696af-9000-472d-a938-3c17b1848d59)

where?

**Reply by Alexsuperfly** [reply]
Ohh I misread your initial as asking for disable invent/equip.
